src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -201,83 +203,86 @@
 					<TextBlock Text="{x:Bind DirectoryPropertiesViewModel.PullInfo, Mode=OneWay}" />
 				</StackPanel>
 			</Button>
+		</StackPanel>
 
-			<Button
-				x:Name="GitBranch"
-				Height="24"
-				Padding="8,0,8,0"
-				Background="Transparent"
-				BorderBrush="Transparent"
-				ToolTipService.ToolTip="{helpers:ResourceString Name=ManageBranches}">
-
-				<Button.Content>
-					<StackPanel Orientation="Horizontal" Spacing="8">
-						<usercontrols:OpacityIcon Style="{StaticResource ColorIconGitBranches}" />
-
-						<!--  Branch Name  -->
-						<TextBlock Text="{x:Bind DirectoryPropertiesViewModel.GitBranchDisplayName, Mode=OneWay}" />
-					</StackPanel>
-				</Button.Content>
+		<!--  Use visibility because it causes a crash to use a TwoWay x:Bind on an element that is inside an element with x:Load (#12589, #12599)  -->
+		<Button
+			x:Name="GitBranch"
+			Grid.Column="2"
```

Comment:
Is there any reason we had to move it out of the StackPanel?

Reply:
StackPanel is loaded dynamically by x:Load. x:Load cannot be used for the branch button, so I took it out from the StackPanel and use Visibility instead.

Reply:
I think the xaml will be more readable if we keep everything in the StackPanel and move the x:Load to the individual buttons.

Reply:
Since each button has a unique x:Load, moving the StackPanel's x:Load to the button would make the condition very complex.